File: 단계별로 풀어보기/1차원 배열/Python/2577.py
# 2577
# 숫자의 개수
# 브론즈2

import sys
from collections import Counter

a = int(sys.stdin.readline())
b = int(sys.stdin.readline())
c = int(sys.stdin.readline())

result = Counter(list(str(a*b*c)))

for i in range(10):
    print(result[str(i)])

File: 단계별로 풀어보기/1차원 배열/C++/2577.cpp
// 2577
// 숫자의 개수
// 브론즈2

#include <stdio.h>

using namespace std;

int main(){

    int a, b, c = 0;
    int result = 0;
    int li[10] = {};

    scanf("%d", &a);
    scanf("%d", &b);
    scanf("%d", &c);

    result = a*b*c;

    while(result != 0){
        li[result%10] += 1;
        result /= 10;
    }

    for (int i = 0; i < 10; i++){
        printf("%d\n", li[i]);
    }
}